advanced/Templates/pihole-FTL-prestart.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env sh
+
+# Source utils.sh for getFTLPIDFile(), getFTLAPIPortFile()
+readonly PI_HOLE_SCRIPT_DIR='/opt/pihole'
+readonly utilsfile="${PI_HOLE_SCRIPT_DIR}/utils.sh"
+# shellcheck disable=SC1090
+. "${utilsfile}"
+
+# Get file paths
+FTL_PID_FILE="$(getFTLPIDFile)"
+FTL_PORT_FILE="$(getFTLAPIPortFile)"
+
+# Touch files to ensure they exist (create if non-existing, preserve if existing)
+mkdir -pm 0755 /run/pihole /var/log/pihole
```

Comment:
How to mute/satisfy this Stickler CI error? It is explicitly wanted here to apply modes only to the deepest directory, of course.

Reply:
You can punch it.
![stickerCI](https://user-images.githubusercontent.com/1385443/190927541-0db8bb24-73fb-4812-be60-cd5b4a607e5e.gif)

Sorry, I don't know.
Stickler is very annoying sometimes.

Reply:
Found it, shellcheck as well: https://www.shellcheck.net/wiki/SC2174